src/geometries/primitives/parametrizedcurve.jl

```diff
@@ -0,0 +1,42 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    ParametrizedCurve(func, range = (0.0, 1.0))
+
+A parametrized curve is a curve defined by a function `func` that maps a parameter `t` to a `Point` in space.
+The parameter `t` is defined in the interval `[a, b]`. The curve can only be evaluated for `t` in the
+interval `[0, 1]`.
```

Comment:
Where is it documented that the coords are always in [0,1]^n? I see that for `BezierCurve` it is stated explicitly in the docstring. What is different for a `ParametrizedCurve` compared to the other geometries is that it already comes with a range for the parameter, which can be something else than [0, 1]. Thus, to avoid confusion I think it would be nice to explicitly state it in the docstring (I would even do it if is documented clearly for all geometries since it is a bit counterintuitive for me).

Reply:
> Where is it documented that the coords are always in [0,1]^n?

The *uv* coords of any geometry are in [0,1]^n by definition. If this feature is not documented yet, then we need to add it to a dedicated section of the Documenter.jl docs instead of repeating it over the docstrings of individual geometries. I am trying to say that this is not the right place to document the fact that Meshes.jl geometries can be called like g(u, v, w, ...). This feature is independent of the construction of the ParametrizedCurve itself.

It is an accident that the docstring of `BezierCurve` mentions it.